modules/installing-troubleshooting-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-troubleshooting-assisted-installer-oci_{context}"]
+= Troubleshooting installation of a cluster on OCI
+
+If you experience issues with using the Assisted Installer to install an {product-title} cluster on {oci-first}, read the following sections to troubleshoot common problems.
+
+[discrete]
+== The Ingress Load Balancer in OCI is not at a healthy status
+
+This issue is classed as a `Warning` as against an `Error`, because by using the Resource Manager to create a stack, you created a pool of compute nodes, 3 by default, that are automatically added as backend listeners for the Ingress Load Balancer. By default, the {product-title} deploys two router pods, which are based on the default values from the {product-title} manifest files. The `Warning` is expected because a mismatch exists with the number of router pods available, two, to run on the three compute nodes. 
```

Comment:
This part is confusing: This issue is classed as a `Warning` as against an `Error`, because . . .

Maybe it needs revised? 

Also, I think it's ok to use either "2" or "two", but you have "3" written as a numeral AND spell it out, and you spell out two each time. Maybe we want to be consistent.

Reply:
I removed the "as against an Error". Does this work for you?